rocketpy/mathutils/function.py

```diff
@@ -369,7 +369,7 @@ def get_value_opt(x):
                 # Interval found... interpolate... or extrapolate
                 if x_min <= x <= x_max:
                     # Interpolate
-                    x_interval = x_interval if x_interval != 0 else 1
+                    x_interval = max(x_interval, 1)
```

Comment:
This line behaves differently than the previous version. Is this correct?

Reply:
It's actually the sane behavior vut without a if statement.

The x_interval will never be negative, it's either 0 or some positive integer value. If it's 0, it should be changed to 1.